### DISCUSSION TITLE
How can I check if the pyodide distribution in my jupyter-lite site is a custom distribution of pyodide?

### DISCUSSION THREAD

**Discussion by savakarrohan** [jupyterlite/jupyterlite#797] [Q&A] [answered]
Self explanatory title but I just tried to build an empty jupyterlite site using the CLI. I was intending to use a custom distribution of pyodide which I downloaded from the most recent github release [pyodide release](https://github.com/pyodide/pyodide/releases)

I saved this tar file in directory `./pyodideDistribution/pyodide-build-0.21.2.tar.bz2` and referenced it during build using the CLI
i.e 
`jupyter lite build --pyodide='./pyodideDistribution/pyodide-build-0.21.2.tar.bz2'` 

##Observation
-I noticed that the build took just a few minutes more.
-The loading time was for the home page was definetly longer.

##My trials
I tried importing numpy and scipy with commands 

_Which were relatively quicker_

```
import numpy as np
import matplotlib as mp
import scipy as sp
```

My doubt is, Is there a method that I can test that the build used the custom distribution of pyodide?

**Comment by bollwyvl** [comment, marked as answer]
The nickle fix is to check the http requests in the browser console to see
if it contacts the CDN at all... Under most circumstances, the mathjax,
pypi and the pyodide ones will be consulted, all on different domains.

Otherwise: it is rather difficult to tell from inside the client UI, as the
python banner will probably be the same.

**Reply by savakarrohan** [reply]
I thought the pyodide one should be taken from the local address and not the CDN.

Is there a difference between pyolite and pyodide? 
The image below shows that pyolite is taken from the local address while libraries such as matplotlib are taken from the CDN.
![image](https://user-images.githubusercontent.com/50200523/189631676-f111e2d7-7aa4-41ec-98d1-031cf1556cac.png)

**Comment by bollwyvl** [comment]
Pyolite is the equivalent of ipykernel (both wrappers of ipython), and
imports pyodide, but pyodide will already be downloaded, set up, and
running before pyolite does anything.

In this case, it looks like you are not getting your custom pyodide as whl
files are coming from jsdelivr.

 What the up in the served /(reply)/jupyer-lite-config.json?

How are you launching the server? It may be there is a bug with the
"jupyter lite serve," debugging server.

**Comment by savakarrohan** [comment]
I had initially tried with the live server extension on vscode,

I now attempted with the command :` jupyter lite serve --pyodide='./pyodideDistribution/pyodide-build-0.21.2.tar.bz2'`

Here is the requests list of the website above.
![image](https://user-images.githubusercontent.com/50200523/190158289-47875c1d-9bdb-4aaf-8195-c4263bdffa5f.png)

I don't see anything with a filename as pyodide or pyolite. 
All the domains are 127.0.0.1:5500.

The website does import numpy and prints hello world.


Further I have attached the `./_output/jupyter-lite-config.json` file below which is json but just uploading purposes.
[jupyter-lite.txt](https://github.com/jupyterlite/jupyterlite/files/9566378/jupyter-lite.txt)
